kong/pdk/node.lua

```diff
@@ -255,6 +257,22 @@ local function new(self)
     return gsub(hostname, "\n$", "")
   end
 
+
+  local prefix = self and self.configuration and self.configuration.prefix
+  if prefix and self.configuration.role == "data_plane" then
+    local id, err = private_node.load_node_id(prefix)
+    if id then
+      node_id = id
+      ngx.log(ngx.INFO, "restored node_id from the filesystem: ", node_id)
```

Comment:
Reduce the verbosity of this log to `DEBUG`, we don't need to print it out every time Kong starts.